drawing_shape2

```diff
@@ -0,0 +1,201 @@
+#!/usr/bin/python3
+# -*- coding: utf-8 -*-
+
+"""
+Fragment kodu odpowiadający za rysowanie z plikow shp metodą lineTo
+
+Problemy:
+    - jak przekazać miejsce kliknięcia myszą do innych funkcji?
+    - jak ponownie wywołać rysowanie skoro już zostało zakończone?
+    - jak zorganizować całosć kodu?
+"""
+
+from PyQt5.QtWidgets import QWidget, QApplication, QGridLayout, QLabel, QMessageBox # QGridLayout, QLabel - do obsługi myszy
+from PyQt5.QtGui import QPainter, QPen, QPainterPath
+from PyQt5.QtCore import Qt
+import sys
+
+class Mapa(QWidget):
+    # czy będzie dobrym pomysłem zdefiniowanie zmiennej klasowej tutaj (przed funkcjami) zawierającej informację o wymiarach okna (w i h)?
+    def __init__(self):
+        super().__init__()
+        self.h = 800    #zdefiniowanie h i w jako zmiennych klasowych (wysokosc i szerokosc okna)
+        self.w = 900
+        self.initUI()
+        
+    def initUI(self):     
+
+        self.setGeometry(150, 150, self.w, self.h-120) #położenie okna poziom, pion, rozmiary okna poziom, pion
```

Comment:
Jeśli chcesz aby z kodu wynikało znaczenie argumentów, to możesz użyć ich nazw. Nie potrzeba wtedy komentarza.